egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -357,11 +385,23 @@ def MakeConfigs(config_dir, splice_indexes_string,
     left_context += int(parsed_splice_output['left_context'])
     right_context += int(parsed_splice_output['right_context'])
 
+    add_lda_str = 'false'
+    if add_lda:
+        add_lda_str = 'true'
+
+    include_log_softmax_str = 'false'
+    if include_log_softmax:
+        include_log_softmax_str = 'true'
+
     # write the files used by other scripts like steps/nnet3/get_egs.sh
     f = open(config_dir + "/vars", "w")
     print('model_left_context=' + str(left_context), file=f)
     print('model_right_context=' + str(right_context), file=f)
     print('num_hidden_layers=' + str(num_hidden_layers), file=f)
+    print('num_targets=' + str(num_targets), file=f)
```

Comment:
@vijayaditya 
I had to add extra options the vars file as they are required by the train script. It seems that the python library to parse the config file currently only supports integer options. Does it make sense to modify it to support generic options (string etc.)?